inference-engine/tests/functional/plugin/shared/src/single_layer_tests/extract_image_patches.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <algorithm>
+#include <functional>
+#include <memory>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "single_layer_tests/extract_image_patches.hpp"
+
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+
+
+namespace LayerTestsDefinitions {
+
+std::string ExtractImagePatchesTest::getTestCaseName(const testing::TestParamInfo<extractImagePatchesTuple> &obj) {
+    std::vector<size_t> inputShape, kernel, strides, rates;
+    ngraph::op::PadType pad_type;
+    InferenceEngine::Precision netPrc;
+    std::string targetName;
+    std::tie(inputShape, kernel, strides, rates, pad_type, netPrc, targetName) = obj.param;
+    std::ostringstream result;
+    result << "IS=" << CommonTestUtils::vec2str(inputShape) << "_";
+    result << "netPRC=" << netPrc.name() << "_";
+    result << "K=" << CommonTestUtils::vec2str(kernel) << "_";
+    result << "S=" << CommonTestUtils::vec2str(strides) << "_";
+    result << "R=" << CommonTestUtils::vec2str(rates) << "_";
+    result << "P=" << pad_type << "_";
+    result << "targetDevice=" << targetName << "_";
```

Comment:
I think we don't need "_" in the end of the test case name.

Reply:
Done